.github/workflows/nightly.yml

```diff
@@ -31,117 +36,157 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: wlan-testing
+        ref: staging-wifi-2047
+
     - name: Checkout LANforge scripts
       uses: actions/checkout@v2
       with:
         path: wlan-lanforge-scripts
         repository: Telecominfraproject/wlan-lanforge-scripts
 
+    - name: import LANforge scripts
+      working-directory: wlan-testing
+      run: ./sync_repos.bash
+
+    - name: create configuration.py
+      working-directory: wlan-testing
+      run: |
+        cat << EOF > configuration.py
+        ${{ secrets.LAB_CONFIGURATION }}
+        EOF
+
+    - name: create Dockerfile
```

Comment:
We have a dockerfile now in docker directory of this repository. Is it possible to use that?